### PR TITLE
Use Ledger API Test Tool for sandbox

### LINKED ISSUE #1371: Conformance tests: drop standalone semantic test runner

### PR DESCRIPTION
**Pull request by gerolf-da**
This PR starts using the ledger api test tool for the sandbox:
1) Run semantic tests and TransactionServiceIT via Ledger API Test Tool
2) Disable TransactionServiceIT in the regular test suite and standalone semantic test runner

Fixes #1371

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-integration-tests/BUILD.bazel [hunk 1]
```diff
@@ -87,7 +87,12 @@ da_scala_test_suite(
     size = "large",
     # WARNING: srcs list has to be explicit here! It can not be added via deps, due to
     # test discovery mechanism in the test_suite macro working only on sources.
-    srcs = glob(["src/test/itsuite/**/*.scala"]),
+    srcs = glob(
+        include = ["src/test/itsuite/**/*.scala"],
+        # the excluded tests are run by the conformance test via the ledger-api-test-tool
+        # see //ledger/sandbox:conformance-test-*
+        exclude = ["src/test/itsuite/**/TransactionServiceIT.scala"],
```
Comment (gleber-da) [inline ledger/ledger-api-integration-tests/BUILD.bazel]: IMO, I feel like keeping it in the IT suite is still worth it as Bazel would do much better caching for those than to these tests via our tool. They are just better for tight iteration cycle.

Comment (gleber-da) [reply ledger/ledger-api-integration-tests/BUILD.bazel]: :+1:

Comment (gerolf-da) [reply ledger/ledger-api-integration-tests/BUILD.bazel]: Having said that, your argument is still valid. And I also have concerns regarding the ergonomics of having to go through the ledger-api-tool to only run a specific integration tests.
For that we could add a bazel target that is not run in CI but still exposes the tests locally.

**Comment by gerolf-da** [comment]
@gaboraranyossy-da: I pushed a commit that allows us to pass extra arguments to the ledger-api-test-tool when running the bazel target.
So instead of:
```
bazel test //ledger/ledger-api-integration-tests:ledger-api-integration-tests_test_suite_src_test_itsuite_scala_com_digitalasset_platform_tests_integration_ledger_api_TransactionServiceIT.scala
```
We can specifically run this test with:
```
bazel run //ledger/sandbox:conformance-test-static-time -- --include TransactionServiceTests
```

**Comment by gerolf-da** [comment]
@gleber-da: I pushed another commit that adds the `all-tests` flag. This allows users to run all default and optional tests with one flag, instead of having to enumerate them one-by-one. WDYT?

**Comment by gleber-da** [comment]
@gerolf-da The flag is great. Please document it too!

**Comment by gerolf-da** [comment]
Sure, already on it. Just wanted to already give you a first look :)

File: release/artifacts.yaml [hunk 2]
```diff
@@ -111,8 +111,6 @@
   type: jar-scala
 - target: //ledger/sandbox:sandbox
   type: jar-scala
-- target: //ledger/ledger-api-integration-tests:semantic-test-runner
-  type: jar-deploy
```
Comment (gerolf-da) [inline release/artifacts.yaml]: @bitonic: I'm removing this semantic tests runner from the release. I don't think anybody was using it anyway, and the preferred method to run this is via the ledger-api-tests-tool.

**Comment by gerolf-da** [comment]
@bethaitman: would you like to look at changes in documentation and release-notes?

**Pull request merged**